plugin/helpers_kv.go

```diff
@@ -89,7 +71,25 @@ func (p *HelpersImpl) KVCompareAndDeleteJSON(key string, oldValue interface{}) (
 	return deleted, nil
 }
 
-// KVSetWithExpiryJSON is a wrapper around KVSetWithExpiry to simplify atomically writing a JSON object with expiry to the key value store.
+// KVGetJSON implements Helpers.KVGetJSON.
```

Comment:
nit: relocated on purpose?

Reply:
Yes, to sync up with the order in the interface